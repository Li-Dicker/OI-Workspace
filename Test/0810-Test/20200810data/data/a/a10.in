10
200
10751 59365 60017 50823 97633 72458 16942 73337 30298 84000 11071 94962 57418 74144 71061 9321 49784 44108 92903 19567 28293 11853 21607 47687 70594 20778 10409 69345 33044 28427 55938 44869 22642 31796 53099 84920 87690 98694 97212 86326 42444 63418 68088 93402 26512 2464 1763 52819 41044 38358 93618 62145 14616 28652 14480 62330 9819 61070 63122 92724 41885 8755 19942 34648 19514 51239 74892 30062 38898 65541 59967 69436 32759 41138 6276 23677 39805 69163 67129 10812 98070 58773 7659 50312 55356 49730 99932 90966 32940 61338 70417 97769 13127 37519 50279 24123 83410 62012 44507 46407 56579 60332 47562 94753 65953 19794 96308 7639 21406 82059 59406 6962 42188 26765 55578 57465 31241 11817 31885 43278 80093 56743 80073 7789 49672 35776 93639 46833 38810 64331 94069 79418 64106 24615 91031 80157 92037 83933 94157 37252 47951 72089 66249 84681 40523 80153 18897 64987 72166 26816 82343 79794 43933 12334 57943 60772 202 4830 71900 16559 36688 33600 9900 18656 2946 50778 3337 64962 13544 56 36046 62076 83075 48753 17732 64251 50563 7594 54314 6767 29468 62545 69085 49703 743 68315 79531 14473 3766 29614 57798 1287 1508 66675 58593 78499 93112 6398 41546 70413 
20443 31457 24834 6840 25195 1056 35893 47185 20333 3374 20276 49276 15205 11561 19115 24038 32146 44793 39284 37888 40925 39704 43444 30123 23755 29435 42172 39777 20025 37057 10901 36398 28160 4976 28541 30423 27358 4598 49980 28397 13475 46408 46490 7828 38409 7556 29493 27970 17034 9879 46517 7353 16190 43696 27101 17753 47869 32675 34549 14110 21927 40502 42505 49831 33507 9096 2657 1267 22630 31835 38112 8451 454 49742 13205 36535 43522 8503 21393 35870 49226 19807 26391 38846 24239 7996 38061 45876 8241 12530 12409 42672 36793 40707 38940 18889 44795 36077 46498 6449 41722 8448 42352 39244 48547 27018 41117 4903 32966 44565 9679 6902 22193 42696 686 29768 46808 16641 48783 37434 1128 33669 26845 5293 14624 37183 17794 8250 7892 37502 17000 39304 49834 49148 20321 25484 36051 34302 8878 38649 26093 34600 15753 44022 40435 15292 18018 16217 26328 47730 33400 4347 16523 9036 38554 45621 13052 40463 13585 48099 18691 43213 19714 18253 15259 32733 20317 31488 24010 35233 23069 29325 159 5652 1691 48460 27073 22950 43931 12830 39721 38692 34721 14746 39633 31252 9627 24556 2427 9357 18735 3109 35479 39261 44627 16282 20431 38027 31309 13662 
198
26955 58921 18942 81556 96742 24163 65159 58023 16640 501 50021 12377 80941 72455 89232 77420 56900 23507 19305 32735 54580 46892 71643 79704 52186 79732 4468 55021 53113 90923 74982 38254 81638 13347 32499 1267 7005 17322 57563 45871 54013 23282 53186 77375 43723 33769 36752 17642 64315 3964 3254 33110 59491 35599 30464 39159 10898 60737 10880 82595 27991 45564 94468 50840 15964 56356 40580 95375 62035 87341 93260 45761 49392 74387 80866 11864 94548 16843 63419 9430 52624 6888 71504 32706 38550 4147 1328 88189 80076 35387 20617 40985 37039 90849 47506 32398 33156 56435 25155 88223 88548 39221 10471 54683 46242 62545 24167 82102 93622 61847 80463 21086 98779 10643 52045 3558 19917 27891 30654 33658 40767 91221 200 40788 79747 7911 87991 87138 56186 98463 29135 60041 88437 25426 32724 53233 64894 42269 92276 67369 19226 90893 17188 37305 65036 4924 92608 32426 87653 38101 82913 34956 97155 56480 1473 5613 9806 61509 17258 45775 53825 88753 73997 5770 55637 21551 37744 3935 52173 89523 81358 2835 62655 49210 86509 90262 95579 33649 57314 55197 4379 84103 76814 24974 28604 48092 89279 77144 62855 70617 53058 50361 45670 76039 28679 580 92057 47293 
37123 18631 40458 16181 9690 14673 35446 26943 36948 28781 19384 22589 21473 17768 44919 30594 47543 7323 15933 9069 4753 3888 48463 24676 27207 2474 19914 6131 2010 22096 28220 34197 10578 14888 29290 140 8741 825 6211 32817 13924 11436 20142 28220 23310 13363 33280 401 570 32475 9080 41981 15030 46443 13875 47521 24379 8741 29114 44765 25478 29227 27150 8594 11850 12667 20012 40983 10504 1625 7979 26305 16883 9178 46571 37477 48638 6307 17247 15099 24646 46294 26506 13530 25777 25944 10927 48053 49770 32734 31542 35691 40478 2568 10979 16214 28457 35896 21205 36044 33549 9613 44782 29589 12682 15924 36540 11498 47013 6010 40582 41691 31450 44031 34001 48335 31375 16977 5066 8779 29143 44229 3346 14214 31452 24276 43097 4737 15175 38456 49309 49919 37124 6581 10713 21059 49501 33495 35828 22122 49091 13976 8345 36486 42270 11336 32886 25800 20050 44150 17949 22904 42447 25397 37888 26614 36262 13781 15145 43590 40871 5274 28437 8223 8728 38186 6409 37011 3498 7506 30608 30489 5848 6845 17416 49258 1552 29348 1373 27056 12768 16510 905 21234 43399 30480 14432 12679 11399 43939 31211 45986 28971 21302 10915 33155 35563 46938 
200
30901 12514 24226 5528 46479 69982 40318 43400 39198 34221 43093 43788 97284 10894 72495 22270 53079 76358 79466 32608 99447 37146 73610 61280 57727 62007 14829 10474 50314 36036 77638 97426 56563 22858 4746 28011 1988 54649 22410 78990 29099 85668 74545 29423 86374 12383 65512 54466 46820 93976 38581 15378 29425 77329 2590 12802 3320 58453 87489 47122 81248 55205 27853 32046 61294 47506 7403 83337 28606 44761 50681 2768 77579 75477 37873 94349 21729 59484 47668 38864 84611 52027 25522 51061 52327 77927 22171 53957 49950 78781 39032 42533 93852 3271 56324 81398 98922 59191 62370 35494 93797 10268 32750 33055 27865 93479 19872 19665 89593 40939 15986 89924 28487 33109 80327 13519 43780 20461 49850 76810 50735 26075 66131 91046 8593 77176 68268 97541 5214 83156 90155 61484 38499 32439 28792 12995 34522 23246 23712 42372 56453 61885 82201 66205 91315 34528 95567 19037 54617 25484 50617 34383 65041 18117 83527 96650 5214 57464 83616 85059 63405 92136 20931 58062 33541 19065 13658 62601 4648 25205 32810 76334 39638 42150 82858 11589 66218 75027 45742 75929 24338 76200 12579 20420 13223 63052 48369 22613 42010 73859 10946 27098 47089 44137 40759 38373 83718 13587 85349 21641 
29186 49987 17615 27934 12325 28509 5317 40638 45767 38302 16892 4925 48197 13648 49106 17436 39778 3113 4888 16413 41225 38980 48589 38248 32772 47260 48748 37257 47664 4264 46801 33245 38109 36320 874 21586 19740 48800 45511 44620 10317 36115 41023 4131 37022 28731 37307 7239 27831 24482 41138 26124 18143 20138 22789 13873 8108 22313 6026 25629 15684 16975 6820 43063 39814 44893 27154 19957 22595 47615 22994 18107 9268 2627 34354 45996 42720 1394 32237 17430 12554 15178 34145 8156 25789 40682 9372 36959 23809 13836 15554 20129 14802 39179 25573 19134 28997 30546 26662 17217 38010 36708 38985 30788 27628 12695 11225 886 25841 39941 23729 36291 26924 40370 43051 32543 38491 25480 5434 25098 3161 34964 8179 21799 18680 34689 39100 20789 3773 15376 20285 3026 39592 10579 4023 26118 430 30147 10787 40893 22073 33717 37524 44195 19752 22853 484 36521 903 35592 28729 20749 24294 48803 17567 12729 4405 45248 42770 1145 26071 32508 22699 45021 7207 38917 34861 44134 23732 8070 14613 44316 3016 34792 30683 12190 35012 34029 18703 30823 26356 44470 32587 5622 43261 1255 35286 25032 13203 37788 575 5415 28142 18154 4786 37674 7952 30332 4456 43827 
196
12203 50660 55021 45403 53528 22017 56321 80351 57569 91907 41110 38576 72847 95877 74832 31770 20109 42880 24544 66669 61187 44083 28147 35199 99625 14626 29827 19490 61123 81223 33158 62510 30074 14828 5207 46822 77706 59869 34469 7848 34525 35580 49907 3254 67862 65044 13092 35101 52388 67085 21115 18289 59205 52298 72454 41984 46391 62698 4104 72260 8320 91802 3465 95713 10538 52709 89818 23545 16209 32430 29333 84577 4816 64782 33186 61844 55505 35774 32798 572 84098 85664 1324 20196 13627 31021 37091 30503 27058 71548 41954 10664 57809 59849 29796 43682 36428 9742 49206 1924 58976 43106 53923 15787 83080 71935 71545 41415 26434 22295 88885 35855 44091 93476 20461 6935 50298 21243 41570 83135 64461 76259 30489 40583 44590 69976 4673 92895 8645 95719 9030 50715 36666 78908 99802 54228 77678 60811 96421 13140 53235 12353 9781 37301 74236 1749 10700 20995 47580 90463 57943 96953 78534 23247 43791 72453 78834 87025 867 41144 44469 92225 16400 89529 3072 81067 14431 79140 75539 91067 53962 5788 62712 51280 12598 78132 5 41472 56736 93561 229 48581 69573 18188 64388 80365 37885 10472 95540 95605 78132 42137 75889 40138 26364 21586 
10083 44796 8442 18641 32035 31187 17384 24545 23708 20034 6632 5499 6287 62 48989 33606 4654 107 45159 21675 6565 24047 46445 29967 24200 13252 30422 6796 30339 12529 27393 10095 5357 14506 23674 23827 628 38517 33932 20987 3552 18363 8236 36912 3252 20788 32006 5013 32879 32626 46449 43535 22275 12735 11348 46469 39173 4956 32718 36873 2187 3495 12768 27417 27580 45950 8724 14765 41241 12757 23329 28729 12272 9873 3404 14931 4994 43810 38659 17374 47409 28745 23641 25963 5907 17075 2756 42870 23404 993 40852 35238 29211 24856 43766 38412 8938 430 40935 37611 23272 16564 14294 12561 34809 38279 44279 43283 13367 35961 1840 11650 43545 34931 15821 39558 1025 20771 47017 28423 12534 34775 21502 18734 43614 47536 9069 8205 31596 20299 2483 24708 6845 14041 23150 21713 38850 39381 18583 14669 33322 31331 46815 25714 12078 2161 41701 24962 49799 38623 14217 7871 28064 8614 33717 23157 43437 1507 6407 11529 10505 29277 19012 31512 2059 30264 23801 31976 18639 5194 47111 48 41826 10868 40940 41226 29661 45556 15528 5890 3722 24682 16870 13935 24563 21456 22454 32581 26696 43580 43117 46278 40250 18344 46118 7598 
197
69688 84412 95908 16081 74985 87916 53498 745 82977 57649 65811 45820 23514 78132 96908 5593 93327 67699 99899 10179 70182 294 51955 21206 62001 85666 14880 27992 88998 18611 97586 71897 13995 72411 17411 19603 44574 89081 55597 83846 4351 77584 51113 94078 39902 20729 27505 13540 84389 5904 31742 6333 78450 84742 81074 86307 89926 65801 47905 46031 15284 54703 39195 43310 94027 96197 30713 81214 67917 40099 54621 32174 22910 39501 35212 59005 58223 54276 53562 85016 84399 21365 27622 16469 16522 63096 59049 89811 80432 6847 9671 79648 4986 4953 16988 96138 42256 16294 764 62581 43634 46189 9387 45338 29444 96675 62325 21065 63120 76754 42553 43496 70174 44960 21159 74310 84477 45394 13370 10131 37565 94623 16447 77978 21540 34340 42784 63730 48756 81898 29871 38391 50591 97876 17723 9947 98370 43133 94759 42649 52254 94120 88466 32209 3836 99630 26221 93805 7200 64819 56818 71465 9474 39772 9327 16667 72497 30985 51530 94137 47699 17384 45140 35849 21872 64230 33388 61773 53247 35545 11193 4220 79199 35444 32762 36806 31118 21310 47037 44820 55364 69236 79038 86691 24535 2102 73511 17142 17913 28777 58424 86261 25780 6413 24550 95175 36344 
35106 47874 14448 45776 44161 14057 7852 42158 24067 14891 11772 11577 40467 11184 43505 11389 39531 49348 39012 16946 34574 24855 25695 34139 16800 14406 12174 32134 22215 15245 16864 30317 7887 39018 36343 17329 48887 12835 21609 1777 8673 17601 12918 12996 9489 44580 40525 35168 38955 22055 19240 1604 30782 5315 7755 40193 20700 12215 10734 24879 19604 41867 44791 45050 39771 2956 24274 12759 30836 19636 9064 4854 23895 41552 5496 8202 25019 28760 36801 8025 26722 45176 3901 29293 30692 3814 36703 14697 20997 9565 5311 49898 6169 4803 47237 3187 30969 18007 45347 12103 17464 39988 44210 49340 30610 49190 37293 5178 7940 28219 10476 29277 10255 18351 18676 10723 46919 28275 49644 8047 7547 33713 49205 39613 31773 14763 37508 9555 49236 30020 11822 27102 11363 34262 33783 5256 10070 9167 7817 1163 7394 49873 44486 43446 26657 22316 45508 23869 20337 13232 29177 34731 26417 41591 17785 19914 41439 26242 25819 11072 12017 79 9737 17771 4105 43872 26612 16070 37271 27696 44837 46208 42568 47220 16971 29493 48887 12945 13194 48374 16680 36961 29644 21675 11227 32504 48819 23226 49144 32713 27173 26398 40184 6805 20006 15871 4517 
197
13899 97943 74716 41499 84184 60989 92406 97578 15066 24115 29258 67035 38442 22703 18393 32743 32800 39674 49799 54645 73331 41813 14545 56074 14822 47201 38841 89807 65140 20991 62502 98141 14856 44312 13980 51005 73633 40910 38174 15896 2988 62651 64531 35284 55796 60518 25444 75344 92906 40453 94693 40926 20746 47627 77892 99736 59003 56500 93620 50064 17482 99405 70340 70755 32511 4341 7997 97133 22561 28380 15951 32151 41278 95200 7067 10313 31199 17405 24574 58174 47108 32157 32538 68168 34855 3973 21522 34522 16605 37427 13984 94083 84614 43603 4174 78835 58527 71107 2815 66628 15029 84060 91907 64543 63520 55062 79306 48500 14979 59803 86053 41356 67541 19418 17596 5280 68388 51983 52556 85148 98326 36541 96103 88115 30102 33281 8022 90701 9361 59117 95341 95394 79195 86680 15834 90530 19678 54109 47769 25976 78919 8367 63755 70303 39349 88798 53255 56842 586 13049 60068 37253 60625 4991 45798 45912 61634 56086 43954 58263 12822 58069 66069 71856 30206 1958 52208 93848 74410 27337 28153 64686 76711 62322 96863 70673 1900 70954 97138 88386 80819 87098 52875 86145 14200 84276 88627 68175 72184 58943 57652 17073 16471 48024 75760 95772 1279 
31443 28363 5769 12106 46705 21432 46141 47021 47037 49080 12773 8867 18283 32471 26048 11924 27593 2913 10532 2524 16390 8604 47268 32454 6828 42402 39476 6789 14064 35210 23150 899 44711 25094 10372 6999 12646 23408 38126 3550 46050 33240 34031 3165 48000 44139 31773 1703 7045 7867 7831 32014 18081 44778 8930 48238 39459 4728 3303 506 48416 39734 24380 18619 39534 3167 27882 1271 8386 21966 39559 15106 34219 24483 32776 15430 44276 742 24150 14943 44666 11196 7213 32237 37897 10095 35443 15206 9585 1875 21060 15968 1446 12109 39391 49428 31522 44590 4944 16922 16449 23934 12302 15763 48564 13805 9831 13906 44087 44633 11892 2586 49157 33186 46724 726 13871 37242 48994 29180 23854 14815 13194 48978 39113 34662 32409 7713 28380 14439 5465 19414 27263 44094 43686 617 5726 17343 13204 26441 16875 7464 28573 37374 16503 9960 8195 27389 43366 999 23495 20416 10654 40031 22229 23266 20833 19394 4936 31029 45717 33723 16789 5583 42005 9620 17618 11674 17890 41905 31301 27462 19967 14353 7741 37482 13013 35272 20371 33223 11298 1286 7668 5085 21818 11612 16914 23587 17047 37633 33616 11714 36423 15820 48008 16058 47987 
198
95258 70100 90639 2290 52052 60428 26749 13258 62214 44172 88226 37169 11281 76575 36883 35162 45305 80400 70727 50207 70812 85059 92875 34991 20424 66934 97623 44908 31615 14278 44200 53205 22240 44659 57742 35274 32856 94894 42342 31507 20609 24782 86841 79305 44175 15245 54974 70647 59115 3804 99389 46969 42080 37964 30550 75757 12906 22248 96066 96560 81258 50355 31794 48615 80264 20879 24970 77129 60864 61747 18842 43069 63067 25468 31952 32567 63052 42018 7066 21188 96217 63450 92435 40170 63827 70299 10688 95189 64239 29448 79381 9672 95939 18821 81334 36589 13829 91789 3504 25161 92505 73878 44909 73571 59636 26961 55233 88710 13727 98710 66163 93361 19426 63935 328 40372 2356 75453 99058 61539 52651 10229 20185 48050 33044 2457 8468 36955 25033 68482 22599 65346 3959 6845 56665 63314 60597 62515 71962 78528 60104 9080 61150 25307 97816 90090 26073 67758 69755 32110 39976 26685 473 4357 45335 19597 57920 33204 81368 25445 92828 78435 27902 93294 20567 47530 44449 42985 3216 81789 13810 92092 13547 49395 66712 58068 56375 77583 90726 34568 66792 38721 892 21070 87634 35226 91319 24262 60722 29986 84591 52973 48699 31063 10840 93297 47917 64736 
26481 30846 32619 8788 6720 11581 29103 46242 12852 47298 11373 4485 25468 40341 41688 24676 20675 40281 23393 41756 22767 27714 19013 9363 44683 45041 24681 9797 11831 15815 28530 49973 22125 34047 18593 88 42222 24666 30514 2973 48280 12092 24080 7219 41503 17270 14350 8950 27505 15938 257 22576 25982 35230 36087 12868 40111 21652 36981 10231 12431 13879 4232 21369 15189 382 47684 11451 14947 24314 17843 27568 8761 10441 13170 36402 49119 5519 27386 3857 26324 2853 35684 4486 320 29876 45528 45966 47937 38225 24533 46775 4702 47930 22648 6506 14827 25713 45632 37365 33011 16087 26791 43148 12415 36171 26048 33069 1277 24433 25787 33881 34919 39652 40576 43725 29761 5772 10313 48339 21443 8213 41673 39445 17470 33519 1182 5405 16057 46294 24314 1996 4551 38619 1008 24559 49108 28268 33277 25536 26081 18492 16657 43555 2117 4264 13391 44804 29320 17785 34096 37599 38775 8881 48942 39301 2999 45477 13501 27736 23217 20403 46727 6070 15636 46056 26194 15275 1072 15022 2609 28307 42030 22574 13310 24594 1889 47784 38452 36005 36092 8337 45668 47198 32864 1986 30321 20845 49416 8657 9778 14216 1672 21878 48214 29934 2634 8098 
197
33177 58069 81319 69490 9237 50199 94945 18601 42187 80151 31123 54852 2807 31816 51561 27758 92476 6727 63931 68211 70906 13913 73555 67409 20063 36180 58574 79157 49031 26464 98742 15353 92160 72184 63834 27109 97754 9902 69941 1706 25999 30666 18930 19101 85366 28861 68005 93255 7625 30925 68926 10769 82340 2773 47465 94662 48506 88377 90300 31894 38920 43944 57023 48703 7718 98902 13075 74083 84642 96589 20515 14822 77961 52577 70729 39007 5901 47189 59073 29857 38812 429 55129 32825 71543 46671 51127 88261 9984 35388 74838 50537 17147 63654 78934 9615 42394 74945 68845 63692 5068 6609 88520 67500 68073 93444 66578 9776 1999 52788 41323 51198 27365 34117 41148 69815 81877 50153 32922 2097 48214 95131 64301 41868 29492 87482 54765 49324 40638 61850 18439 11862 5550 47635 35589 92407 81030 48290 66771 77458 96551 88512 42647 20086 94846 60515 58605 8028 71418 94623 60971 4743 57783 6967 92710 61811 36430 43830 84457 76644 2802 66920 5037 40788 52342 14337 65748 62947 12843 66359 15223 7853 22078 22043 9009 55212 79190 65731 88282 14006 14615 19092 52566 15626 47959 92194 84469 15452 52655 32908 59998 52927 40570 50009 84849 25205 28090 
11838 22917 5190 21958 3595 10682 40853 6763 22111 27593 32073 1298 17399 19836 31132 39052 14077 35873 20350 18569 46369 68 17811 7598 9378 29162 20022 30100 45174 37937 25844 5839 41584 45401 33020 6740 20400 6587 31400 40994 49621 14619 25612 757 43672 36050 20826 5167 30556 14503 9092 42329 39219 7808 18270 15989 1625 36150 11487 26516 3520 34345 37039 44188 9251 20710 11359 8386 6712 642 43046 42616 28655 32792 15424 39937 46687 6473 23412 7249 4221 14287 28837 21449 45692 35807 3080 26473 5015 40875 4536 5643 7546 44335 31962 9067 4427 41376 16368 44228 400 16406 14350 44505 4672 16348 47655 15508 31187 40189 33322 29184 48822 28454 25370 10171 18470 36603 5264 2270 3284 39319 21178 7250 12689 46616 2628 924 33953 48205 16576 25439 9662 879 35268 9161 43454 41949 9126 44470 26776 1068 40149 17627 23594 25369 6921 26552 44729 25125 26522 30048 23961 15022 41000 2147 8881 15108 48957 46762 4647 15076 25725 6933 25484 27619 13290 2232 10808 6247 45986 26609 24082 46496 22792 33045 6404 48260 36509 34134 30269 42863 21924 45078 11781 34319 6357 6632 17097 23705 33312 29791 20236 11077 13762 23303 38832 
197
21589 84213 84909 98739 22571 78860 36897 97505 16517 56785 65764 28323 58667 90542 97087 5114 97188 85428 74589 83903 32934 51882 94432 82367 18201 77226 27252 38392 64135 4570 12664 3373 23364 6962 40811 59334 29961 70160 84501 35566 7827 98576 44329 63373 87055 9610 833 64742 33026 60334 67801 89252 64241 70258 50938 73135 42606 18350 66485 79273 84247 96670 54317 41833 99254 25694 35853 38797 7277 81705 33675 17050 19919 56420 35922 42045 31765 54750 58562 20947 47109 60951 85220 8463 70747 13481 3182 22977 53005 17181 28753 69014 85359 91845 144 1086 45221 51424 21136 73619 87237 36433 65521 88691 3707 63501 51642 55768 44324 38287 68483 78780 14203 70326 51678 15612 91682 40380 79969 17910 13569 44471 2262 5542 25495 41706 37267 94363 50986 45989 72572 85593 12205 3050 79209 20337 2156 451 84512 86332 22023 78824 53621 89058 32209 61871 51284 11283 44828 27871 21430 6575 4862 90762 89917 11615 46290 43643 57380 16821 14207 53172 11856 19629 9843 35583 62577 94872 33412 97078 11166 94080 83620 54221 87840 63571 5301 15273 82842 46902 34952 5393 51314 76669 18748 23428 73166 21304 4085 82035 40862 52303 86764 64216 45989 30292 94794 
35376 43958 47485 27235 12940 42334 16690 35959 32412 14432 35829 21956 10383 1399 16317 13121 37222 11828 34560 40391 12300 37331 39612 2532 33329 8421 43541 33481 37429 21508 31152 27163 41215 2311 30705 43123 14946 38371 18753 2340 34966 47367 33348 111 36558 9180 4412 873 40491 13252 26181 358 14456 33400 6875 47979 47577 10289 32256 24883 37091 30448 18023 39467 28233 4570 42671 21888 26506 28622 8510 41766 34858 34889 34858 25713 33123 39618 35287 38159 5886 40325 8391 10623 7468 29894 39541 3213 1536 11587 28163 42202 20168 14120 1201 48031 43874 20068 3616 15987 47140 45521 45854 32204 24381 41183 43306 39832 35096 49644 7728 14979 39481 6842 13453 870 49483 641 26758 18953 18126 24079 19370 49252 38240 38086 19174 9418 5905 3998 15433 34032 28088 37010 48866 34372 2314 44523 9915 36628 23030 28934 15735 32412 438 34561 1554 24228 45505 28749 33908 49006 35824 22250 43684 28475 41159 35115 2798 34705 19701 22579 42123 20547 47099 1057 24320 7586 3019 26325 48354 33707 16410 44050 34680 3926 5828 15228 15676 47963 24029 38044 9442 31340 49317 30283 39212 32637 18084 14533 45376 804 27866 34122 46294 10729 30250 
200
4528 15200 78602 27338 27340 39484 87522 60387 39555 63217 78848 74311 35835 15347 72389 75867 46746 31852 45315 35971 83574 21248 28205 67341 52779 9785 75750 66682 12374 42859 360 76963 61785 62753 88917 85738 63556 30024 39479 97269 13665 73313 91321 60045 48780 17231 95851 60563 75158 40363 92651 52304 21868 13017 93671 93856 84675 60093 57327 17152 51312 46486 27172 72119 51824 80937 67894 13218 55940 65097 15574 38780 69430 58513 31990 95883 75782 51829 18483 60641 70185 45010 18010 6687 40473 71646 97623 79457 81585 7998 94891 92123 15786 7184 56540 39163 46409 83669 36657 48453 25785 31263 50282 97933 77653 33452 24368 96887 42215 70273 85619 89834 10107 66224 49003 49367 1643 59105 18876 10888 55279 48625 20143 75326 68789 54895 74754 32793 63105 7982 32694 54927 9080 26414 33993 738 59996 9873 44517 27260 19269 84674 86667 17773 18987 13087 29642 48646 87391 19310 87105 56823 38976 29619 81185 78688 63644 72081 44976 63604 35849 2257 73749 41924 25436 74939 1197 61908 90641 35180 10887 65922 37377 13397 63596 56513 44019 45540 25364 87539 22142 79787 18951 71236 46139 11638 6356 54854 96112 24827 86267 19059 83886 1847 97308 33405 48667 41961 43966 7449 
205 29637 42988 19787 23024 49793 18607 41016 15319 44868 41102 22436 30174 26739 19555 20367 20666 9114 14059 4148 1139 6392 35156 15957 42586 46596 731 3772 4961 8836 25063 41619 30426 27377 1903 22200 13994 31757 1876 23631 22301 17092 42367 13819 25950 3550 32553 20681 39659 12436 31575 17526 5298 18492 441 48413 4458 11134 27096 34622 8046 31920 19057 16574 25759 41299 45327 7843 12520 19192 43891 16037 31111 35887 3224 38217 23748 26726 9876 40405 26836 16822 16500 6765 30772 15423 9066 24678 35002 8502 41125 425 45548 2455 8559 25989 37174 17388 23218 24085 42693 26692 14896 21797 42356 26524 7739 35034 413 43266 47994 26507 43167 39301 49475 35794 11979 15592 36508 6944 32455 22600 40328 3705 6008 35373 20383 11341 17034 4717 37213 15481 20975 4707 19347 21030 28390 8068 16535 12303 7935 6379 43067 30515 21689 6505 23527 9167 5080 6312 11665 14980 38799 10770 18792 48704 23511 22160 15834 37555 5591 13386 29668 24587 38692 17834 33725 6249 41241 49608 49563 37382 23160 3765 11912 14931 1654 12002 20665 28068 3796 43574 25828 9558 28805 39682 16095 33126 2134 30865 43494 6828 37975 17635 20869 31841 30958 989 25179 42041 
