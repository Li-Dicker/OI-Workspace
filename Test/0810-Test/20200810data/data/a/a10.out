Case #1: 14439569
Case #2: 13856020
Case #3: 13890493
Case #4: 12999309
Case #5: 14022855
Case #6: 13739178
Case #7: 13795625
Case #8: 13204247
Case #9: 13828999
Case #10: 13615308
